src/goFormat.ts

```diff
@@ -23,30 +24,43 @@ export class GoDocumentFormattingEditProvider implements vscode.DocumentFormatti
 		let formatTool = goConfig['formatTool'] || 'goreturns';
```

Comment:
remove this because `getFormatTool` does this.

Reply:
:+1: